docs/transforms.md

```diff
@@ -0,0 +1,84 @@
+## [\[source\]](../fastai/transforms.js)
+
+## Classes
+
+* <a href="normalize">Normalize</a>
+* <a href="normalize">Denormalize</a>
+
+#### <a href="#normalize">class Normalize</a>
+Normalizes an image
+  * code
```

Comment:
I don't think we should put the source in the docs - it's easy enough for the user to jump to the source themselves